blob/commitment_proof.go

```diff
@@ -33,8 +32,8 @@ type CommitmentProof struct {
 	NamespaceID namespace.ID `json:"namespace_id"`
 	// RowProof is the proof of the rows containing the blob's data to the
 	// data root.
-	RowProof         coretypes.RowProof `json:"row_proof"`
-	NamespaceVersion uint8              `json:"namespace_version"`
+	RowProof         proof.RowProof `json:"row_proof"`
```

Comment:
Is it possible/trivial to put RowProof and its methods in go-square? Because it would unlock celestia-opernrpc having the validation methods

Reply:
cc: @rach-id

Reply:
putting that in go-square would require core to depend on go-square. Is that something we want to do?

Reply:
idk if it's something we want to do. Can we please take this discussion outside of this PR b/c IMO shouldn't block this PR. https://github.com/celestiaorg/go-square/issues/100